app/controllers/favorites_controller.rb

```diff
@@ -1,7 +1,7 @@
 class FavoritesController < ApplicationController
 
   def index
-    @fav_pets = FavoritesList.pets
+    @fav_pets = Pet.where(id: cart.contents.keys.map {|key| key.to_i })
```

Comment:
I wonder if there is a more refined way in which we can achieve this same information? Great job getting these into an array of pet_ids